### ISSUE TITLE
Suggestion - override the default margin on body

### ISSUE THREAD

**Issue by jzhu01** [HarringtonR/prj-rev-bwfs-tea-cozy#2]
Seen here: 
https://github.com/HarringtonR/prj-rev-bwfs-tea-cozy/blob/2b5ccc6518805a8956935b925ade9361388e8702/TEA/css/style.css#L1-L3
Since this is 8px by default, you'll want to override it so that your header looks as though it's anchored to the top of the page.

**Comment by HarringtonR** [comment]
yeah i couldn't figure that out... again  sorry not sure what you mean by override.... :(

**Comment by jzhu01** [comment]
Sure! No worries, you just want to set it as margin: 0px.
